src/pages/SSLModule/detail/index.tsx

```diff
@@ -63,41 +82,43 @@ const Detail: React.FC = () => {
             label="SNI"
             name="sni"
             rules={[
-              { required: true, message: formatMessage({ id: 'component.ssl.fieldSNIInvalid' }) },
+              {required: true, message: formatMessage({id: 'component.ssl.fieldSNIInvalid'})},
             ]}
           >
-            <Input />
+            <Input/>
```

Comment:
Please keep the needed space.

Reply:
Oh, Ctrl+Alt+L habitual, it automatically formats the code. 🤣